.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-<!--
-Thank you for contributing to Snowplow!
+/!\ IMPORTANT /!\
```

Comment:
Is it going to be bold?

Reply:
No :smile: I'm updating with the new templating of GH